src/metriculous/evaluators/_classification_figures_bokeh.py

```diff
@@ -141,7 +141,7 @@ def _bokeh_confusion_matrix(
     """
 
     def figure() -> Figure:
-        cm = sklmetrics.confusion_matrix(y_true, y_pred)
+        cm = sklmetrics.confusion_matrix(y_true, y_pred, labels=list(range(y_true.max() + 1)))
```

Comment:
Hi Magnus : ) Thanks for the PR!

I'm wondering if it would be even safer to do

```suggestion
        cm = sklmetrics.confusion_matrix(y_true, y_pred, labels=list(range(len(class_names)))
```

That would also cover the case where `y_true` doesn't contain all the classes.

Reply:
fair point ill buy that! :D

Reply:
@magnusja I've committed this fix is to develop (4246450a) to get it in without the other changes. Thanks again!

Reply:
Ok perfect, sorry totally forgot about this I guess then we can close this